pasqal_cloud/__init__.py

```diff
@@ -49,13 +50,26 @@ def __init__(
         # Ticket (#622), to be removed, used to avoid a breaking change during the group to project renaming
         if not project_id:
             if not group_id:
-                raise TypeError("Either a group_id or project_id has to be given as argument")
-            warn('The parameter group_id is deprecated, from now use project_id instead',
+                raise TypeError("Either a 'group_id' or 'project_id' has to be given as argument")
+            warn("The parameter 'group_id' is deprecated, from now use 'project_id' instead",
                  DeprecationWarning,
                  stacklevel=2
                  )
             project_id = group_id
 
+        # TBR Ticket slot
+        if endpoints or auth0:
+            warn("The parameters 'endpoints' and 'auth0' are deprecated, from now use 'env' instead",
+                 DeprecationWarning,
+                 stacklevel=2
+                 )
+
+        if env:
+            # TBR Ticket slot
+            if endpoints or auth0:
+                raise TypeError("Either 'endpoints' with 'auth0' OR env parameters must be used, not all together")
```

Comment:
The message makes it hard to understand the expected logic, can you reword slightly? I'm not actually sure what the limitation is from this alone

Reply:
Basically it's to make sure that the user will understand the endpoints the sdk will use as if he provides 'endpoints', 'auth0' and 'env' he won't know which one will be used. This is here to prevent confusion